quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/QuarkusPropertiesDistTest.java

```diff
@@ -84,20 +84,20 @@ void testIgnoreQuarkusPropertyFromKeycloakConf(LaunchResult result) {
         cliResult.assertBuild();
     }
 
-    public static class DisableConsoleLogHandler implements Consumer<KeycloakDistribution> {
+    public static class UpdateConsoleLogLevelToWarn implements Consumer<KeycloakDistribution> {
```

Comment:
Isn't this code duplicated from `QuarkusPropertiesAutoBuildDistTest.java` should it go to an `abstract` common ancestor?

Reply:
I didn't want to invest too much time, tbh. As there are plans to remove the "all quarkus raw properties are buildtime options for keycloak" concept, these tests would need a total revamp nevertheless.